sound/soc/sof/control.c

```diff
@@ -369,6 +369,14 @@ int snd_sof_bytes_ext_volatile_get(struct snd_kcontrol *kcontrol, unsigned int _
 	int ret;
 	int err;
 
+	/*
+	 * Decrement the limit by ext bytes header size to
+	 * ensure the user space buffer is not exceeded.
+	 */
+	if (size < sizeof(const struct snd_ctl_tlv))
+		return -EINVAL;
+	size -= sizeof(const struct snd_ctl_tlv);
```

Comment:
mmh, I'll go back to school if anyone shows to me, that `sizeof(const struct x)` can be different from `sizeof(struct x)` :-) Maybe we don't need `const` above

Reply:
fixed